frontend_react/src/shared/ui/components/custom/small-card.tsx

```diff
@@ -0,0 +1,78 @@
+import { Check, Edit, MoreHorizontal } from 'lucide-react';
+
+export type SmallCardProps = {
+  title: string;
+  limit: string;
+};
+
+export const SmallCard = ({ title, limit }: SmallCardProps) => {
+  const BORDER_LENGTH: number = 25;
+  const truncatedTitle: string =
+    title.length > BORDER_LENGTH
+      ? title.slice(0, BORDER_LENGTH) + '...'
+      : title;
+
+  const edit = () => {
```

Comment:
コンポーネントに渡す系は、命名`handleEdit`とかが多いかも
イベントを引数として受け取って、処理する

関数はレンダリングのたびに違うオブジェクトとして生成されるから、useCallbackでメモ化するのが良い

```suggestion
  const handleEdit: React.MouseEventHandler<HTMLButtonElement> = useCallback(() => {
```